### ISSUE TITLE
Some questions about loss function

### ISSUE THREAD

**Issue by Elizbeth-zhou** [MIVRC/MLEFGN-PyTorch#2]
I see you have many loss function can be chosen,like adversarial loss and vgg loss.

And you also said it's not a GAN-based model,does it means if I use the adversarial loss,the original model MLEFGN can be seen as the '**generator**',it generates the image to the discrimator.

And does the vgg loss means the perceptual loss ? But i didn't seen any pretrained model like vgg-19.

Thanks !

**Comment by CV-JunchengLi** [comment]
Many loss functions are defined in the file, but they are not used.
If you want to use them, you need to do some processing yourself.
If you use the adversarial loss, the original model MLEFGN can be seen as the 'generator', it right.
When you use the vgg loss,  the code will download the pre-trained model file. 
This code is adapted from EDSR.
For more information, please refer to https://github.com/thstkdgus35/EDSR-PyTorch.

**Comment by Elizbeth-zhou** [comment]
Thanks
